tests_e2e/orchestrator/scripts/waagent-version

```diff
@@ -16,23 +16,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-# The script is needed for agent-update tests to find agent python for updating agent version in source file versoon.py
+# returns the version of the agent
 #
 set -euo pipefail
 
-PYTHON=""
-# Looking for Agent's Python
-shebang=`cat $(which waagent) | head -1 | grep '^#!'`
-if [[ -z ${shebang+x} ]]; then
-    echo "ERROR: Can't determine Agent's Python."
-    exit 1
-fi
-shebang=`echo $shebang | sed 's/^#!//'`
-# example /usr/bin/env python3
-# some distros will have like /usr/bin/python3.6
-read -ra strarr <<< "$shebang"
-for val in "${strarr[@]}"; do
-  PYTHON=$val
-done
-echo $PYTHON
+python=$(get-agent-python)
+waagent=$(get-agent-bin-path)
+echo "Agent's path: $waagent"
```

Comment:
I suggest removing this echo so that the output is just the same as "waagent --version"